app/scripts/services/Timer.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+  function Timer($interval) {
+    var Timer = {};
+    var stop;
+
+    Timer.counter = 1500;
```

Comment:
Consider placing `1500` in a variable that can be used every time you set the time to `1500` so that if it changes you don't have to change it everywhere.